Code/Framework/AzToolsFramework/AzToolsFramework/Entity/PrefabEditorEntityOwnershipService.cpp

```diff
@@ -67,30 +66,18 @@ namespace AzToolsFramework
     void PrefabEditorEntityOwnershipService::Destroy()
     {
         StopPlayInEditor();
+        Reset();
+
         m_editorSliceOwnershipService.BusDisconnect();
         m_sliceOwnershipService.BusDisconnect();
-
```

Comment:
Any reason for changing the order of Reset() now? Unless this fixes something, I recommend keeping the old order to avoid running into unknown issues.

Reply:
yes this is a mistake I think, just messed up during refactoring. Typically buses should be disconnected prior to deleting anything.